src/lib/SapphireClient.ts

```diff
@@ -32,7 +32,7 @@ export class SapphireClient extends Client {
 	public logger: ILogger;
 
 	/**
-	 * The commands the framework has registered.
+	 * The commands arguments the framework has registered.
```

Comment:
Change 'commands' to 'command'

Reply:
[another thought](https://github.com/sapphire-project/framework/pull/44#discussion_r479311638)